explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -73,16 +98,34 @@ object AladinCell extends ModelOptics {
       // avoids us needing a ref to a Fn component
       .useStateView(false)
       // Listen on web worker for messages with catalog candidates
-      .useStreamOnMountBy((props, _, _, _) =>
-        props.ctx.worker.stream
-          .map(decodeFromTransferable[CatalogResults])
-          .unNone
-          .map(_.candidates)
+      .useStreamBy((props, _, _, _) => (props.target.get, props.obsConf.map(_.obsInstant)))(
```

Comment:
Just curious. What prompted the change from `useStreamOnMountBy` to `useStreamBy`.